core/templates/base.html

```diff
@@ -2,6 +2,7 @@
 {% load compress %}
 {% load replace %}
 {% load staticfiles %}
+{% load pipeline %}
 
```

Comment:
put django-pipeline configurations back into the application to compress CSS/JSS into individual files